partiql-value/src/lib.rs

```diff
@@ -317,6 +317,8 @@ pub trait NullableOrd {
 impl NullableEq for Value {
     type Output = Self;
 
+    // TODO: `eq` and `neq` are not quite right as implemented. Should be able to assert equality between
+    //  different comparable types (e.g. numerics)
```

Comment:
I found that equality wasn't implemented exactly right for the edge case of comparing different comparable types (e.g. `1` = `1.0`). Marked as a TODO here.